src/re_frame/subs.cljc

```diff
@@ -123,7 +124,9 @@
          (if (nil? handler-fn)
            (do (trace/merge-trace! {:error true})
                (console :error (str "re-frame: no subscription handler registered for: \"" query-id "\". Returning a nil subscription.")))
-           (cache-and-return query [] (handler-fn app-db query)))))))
+           (let [sub-reaction (handler-fn app-db query)]
+             (do (trace/merge-trace! {:tags {:value (deref sub-reaction)}})
```

Comment:
Hmm, I just realised that this will cause the subscription to run and be calculated immediately, rather than returning a reaction. I think it would be better to only capture the subscription value in `:sub/run`, not `:sub/create` as this could lead to (or mask) subtle bugs. Concretely, can you pull out the trace here and on line 116? We'll keep it in on the :sub/run. Sorry for not picking this up earlier.